Data/Csv.hs

```diff
@@ -140,24 +143,33 @@ import Data.Csv.Types
 -- Derived:
 --
 -- > {-# LANGUAGE DeriveGeneric #-}
+-- > {-# LANGUAGE OverloadedStrings #-}
+-- >
+-- > import Data.Text    (Text)
+-- > import GHC.Generics (Generic)
 -- >
 -- > data Person = Person { name :: !Text , salary :: !Int }
--- >     deriving Generic
+-- >     deriving (Generic, Show)
 -- >
 -- > instance FromRecord Person
 -- > instance ToRecord Person
 --
 -- Manually defined:
 --
+-- > {-# LANGUAGE OverloadedStrings #-}
```

Comment:
`deriving Show` is for sake of `decode`. Without it `ghci` throws exception:
`No instance for (Show Person) arising from a use of ‘print’`

`OverloadedStrings` will be removed